src/site/shared/containers/Header/Right/UtilitiesDropdown.tsx

```diff
@@ -27,7 +29,8 @@ export const UtilitiesDropdown = ({ extraUtilities }: Props) => {
               onClose={() => dispatch(CloseHeaderMenus())}
               btnInfo={{title: "Utilities", src: "img/icons/utilities.svg"}}>
         {extraUtilities.map(utility => (
-            <div key={utility.popupName} onClick={() => { dispatch(CloseHeaderMenus()); dispatch(OpenHeaderPopup(utility.popupName)); }}>
+            <div key={utility.popupName} onClick={() => {!info.locked && dispatch(CloseHeaderMenus()); 
+            !info.locked && dispatch(OpenHeaderPopup(utility.popupName)); }}>
```

Comment:
nit: formatting, should expand it out since the expression is a bit more complex
```tsx
<div key={utility.popupName} 
     onClick={() => {
         if (info.locked)
             return;
         dispatch(CloseHeaderMenus()); 
         dispatch(OpenHeaderPopup(utility.popupName)); 
     }}>
```